### DISCUSSION TITLE
How to build a loop in a flow?

### DISCUSSION THREAD

**Discussion by mlingelbach** [postmanlabs/postman-flows#96] [Q&A]
Hi!

I have a request and depending on the response, the flow should continue or resend the request.
For this, I use the Validation (true = continue, false = do it again, til the response has the expected value).
True works fine, but I don't know how I can link "false" with the request.
Can you help please?

Thanks,
Markus

**Comment by samay-v** [comment]
@mlingelbach  This can be achieved by connecting the false port of the validate block to the input port of the request. Like such 

![image](https://user-images.githubusercontent.com/28357106/141185314-cc5705b0-f583-4b1b-b287-8ff1ea58c3a4.png)

 2 things to consider here:
1. Be careful of an infinite loop 👀 
2. If you we're passing variables to the request, you might have to use the merge block before the request itself.
